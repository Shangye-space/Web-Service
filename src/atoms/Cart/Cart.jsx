import React from 'react';

import styles from './Cart.module.scss';

const Cart = () => {
    return (
        <div className={styles.cartWrapper}>
            <a className="fas fa-shopping-cart"></a>
        </div>
    );
};

export default Cart;
