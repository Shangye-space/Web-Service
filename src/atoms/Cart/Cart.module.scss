@import '../../settings/variables.scss';

.cartWrapper {
    display: block;
    color: $light-grey;

    font-size: 26px;

    & :hover {
        color: $soft-orange;
        cursor: pointer;
    }
}
