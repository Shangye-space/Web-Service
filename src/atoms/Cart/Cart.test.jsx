import React from 'react';
import ReactDOM from 'react-dom';
import Cart from './Cart.jsx';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Cart />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<Cart />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
