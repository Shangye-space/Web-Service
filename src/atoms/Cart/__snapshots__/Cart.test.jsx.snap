// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="cartWrapper"
>
  <a
    className="fas fa-shopping-cart"
  />
</div>
`;
